Themes/default/css/index.css

```diff
@@ -392,12 +385,12 @@ blockquote cite::before {
 	font-size: 1em;
 }
 /* Styling for BBC tags */
-.bbc_link:link, .bbc_link:visited {
+.bbc_link {
 	border-bottom: 1px solid #A8B6CF;
 }
 .bbc_link:hover {
 	text-decoration: none;
-	border-bottom: 1px solid #346;
+	border-bottom-color: #346;
```

Comment:
changes like this requires consistency, either go for all CSS or just revert it.